config/config.go

```diff
@@ -39,6 +40,8 @@ const (
 	SSHHostCertEndpoint = "/sig/ssh-host-cert"
 	// BlobEndpoint specifies the endpoint for raw signing.
 	BlobEndpoint = "/sig/blob"
+	// DefaultPKCS11Timeout specifies the max time required by HSM to sign a cert.
+	DefaultPKCS11Timeout = 5 * time.Second
```

Comment:
can we make it configurable? Maybe in a follow-up PR.

Reply:
Yes once we are confident of the fix will make it configurable.